client/src/pages/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React, { useState } from 'react';
+import {
+  Button,
+  CssBaseline,
+  TextField,
+  Grid,
+  FormControl,
+  MenuItem,
+  Select,
+  Typography,
+  Container,
+  Paper,
+  Box,
+} from '@material-ui/core';
+import DateFnsUtils from '@date-io/date-fns';
+import useStyles from './useStyles';
+import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+
+export default function EditProfile(): JSX.Element {
+  const classes = useStyles();
+  const [selectedDate, setSelectedDate] = useState(new Date());
+  const [gender, setGender] = useState('male');
+  const optionSelect = [];
+  for (let i = 1; i < 32; i++) {
+    optionSelect.push(
+      <MenuItem value={i} key={i}>
+        {i}
+      </MenuItem>,
+    );
+  }
+
+  return (
+    <Container component="main" className={classes.main}>
+      <CssBaseline />
+      <Grid container>
+        <Grid item md={2} sm={12}>
+          <Typography className={classes.typo} variant="h6">
+            Edit Profile
+          </Typography>
+          <Typography className={classes.typo}>Profile Photo</Typography>
+          <Typography className={classes.typo}>Availability</Typography>
+          <Typography className={classes.typo}>Payment</Typography>
+          <Typography className={classes.typo}>Security</Typography>
+          <Typography className={classes.typo}>Settings</Typography>
+        </Grid>
+        <Grid item md={8} sm={12}>
+          <Paper>
+            <Typography variant="h4" className={classes.title}>
+              Edit Profile
+            </Typography>
+            <form className={classes.form} noValidate>
+              <Box className={classes.box}>
+                <label htmlFor="first-name" className={classes.label}>
+                  FIRST NAME
+                </label>
+                <TextField type="text" id="first-name" placeholder="john" variant="outlined" style={{ width: '70%' }} />
+              </Box>
+              <Box className={classes.box}>
+                <label htmlFor="last-name" className={classes.label}>
+                  LAST NAME
```

Comment:
Some of this code is from the previous PR, please adjust the base of your current PR so that only the difference is visible.